src/librbd/io/ImageRequest.h

```diff
@@ -107,6 +116,7 @@ class ImageRequest {
   Extents m_image_extents;
   ZTracer::Trace m_trace;
   bool m_bypass_image_cache = false;
+  std::atomic<bool> throttle_flag = { true };
```

Comment:
Nit: rename to ```m_throttled```. Also -- no need to use atomic since it won't ever be hit by concurrent threads.

Reply:
okey. , actually, it's for future read throttle and write throttle. then we need to update the flag for each of them. But you are right, no need for now. thanx